airbyte-integrations/connectors/source-okta/source_okta/source.py

```diff
@@ -205,7 +205,7 @@ def path(self, **kwargs) -> str:
 
 
 class CustomRoles(OktaStream):
-    primary_key = "id"
+    primary_key = "label"
```

Comment:
Why this change here?

Reply:
good catch, that was a mistake. I corrected it.